deis-tests/manifests/deis-tests-pod.yaml

```diff
@@ -10,5 +10,5 @@ spec:
   - name: deis-e2e
     image: quay.io/deisci/deis-e2e:canary
     command: ["/bin/sh", "-c"]
-    args: ["/bin/tests.test || true"]
+    args: ["/bin/tests.test"]
```

Comment:
@mboersma I'm not sure why this was necessary to have... and it prevents us from getting the right exit code from the ginkgo test run...